test/module_definition/parameter_documentation_test.py

```diff
@@ -34,3 +37,112 @@ def test_illegal_formatting_fails(text):
     except ValueError:
         return
     assert False
+
+
+def test_duplicate_in_out_attributions_are_not_allowed():
+    # given:
+    text = '@param[in, out] test'
+
+    # when:
+    try:
+        ParameterDocumentation.from_param_string(text)
+    except ValueError:
+        return
+    assert False
+
+
+@pytest.mark.parametrize('text,fixed_length', [
+    ('@param[in, fixed-length=3] test', 3),
+    ('@param[in, fixed-length=2] test', 2),
+    ('@param[in,fixed-length=5] test', 5),
+    ('@param[in,fixed-length =8] test', 8),
+    ('@param[in,fixed-length= 9] test', 9),
+])
+def test_parse_fixed_length(text, fixed_length):
```

Comment:
These are rather tests that check whether the common parsing logic of `ActiveAttributions` works. Make these tests of that class and leave only one or two that check whether the value is correctly set in `ParameterDocumentation`.